nms/app/packages/magmalte/app/components/admin/AuditLog.js

```diff
@@ -13,157 +13,119 @@
   const {response, error, isLoading} = useAxios({
     url: '/admin/auditlog/async',
     method: 'get',
   });
+  const [currRow, setCurrRow] = useState<AuditLogRowType>({});
+  const onClose = () => setJsonDialog(false);
+  const [jsonDialog, setJsonDialog] = useState(false);
 
   if (error || isLoading || !response || !response.data) {
     return <LoadingFiller />;
   }
-
-  const rows = response.data.map(log => (
-    <TableRow key={log.id}>
-      <TableCell>{log.actingUserId}</TableCell>
-      <TableCell>
-        <DeviceStatusCircle
-          isGrey={false}
-          isActive={log.status === 'SUCCESS'}
-        />
-        {log.mutationType}
-      </TableCell>
-      <TableCell>{log.objectType}</TableCell>
-      <TableCell>{log.objectId}</TableCell>
-      <TableCell className={classes.actionsCell}>
-        <NestedRouteLink to={`/details/${log.id}`}>
-          <IconButton>
-            <MoreVert />
-          </IconButton>
-        </NestedRouteLink>
-      </TableCell>
-    </TableRow>
-  ));
-
   return (
-    <>
-      <div className={classes.paper}>
-        <div className={classes.header}>
-          <Text variant="h5">Audit Log</Text>
-        </div>
-        <Paper elevation={2}>
-          <Table>
-            <TableHead>
-              <TableRow>
-                <TableCell>User</TableCell>
-                <TableCell>Action</TableCell>
-                <TableCell>Object Type</TableCell>
-                <TableCell>Object ID</TableCell>
-                <TableCell className={classes.actionsColumn} />
-              </TableRow>
-            </TableHead>
-            <TableBody>{rows}</TableBody>
-          </Table>
-        </Paper>
-      </div>
-      <Route
-        path={relativePath('/details/:id')}
-        render={routeProps => (
-          <JSONDialog
-            json={
-              response.data.find(log => log.id == routeProps.match.params.id)
-                .mutationData
-            }
-            onClose={() => history.push(relativeUrl(''))}
+    <div className={classes.dashboardRoot}>
+      <Grid container spacing={4}>
+        <Grid item xs={12}>
+          <CardTitleRow
+            icon={ListAltIcon}
+            label={`Audit Logs (${response?.data?.length ?? 0})`}
+          />
+        </Grid>
+        <Grid item xs={12}>
+          <JsonDialog open={jsonDialog} onClose={onClose} row={currRow} />
+          <ActionTable
+            data={response.data}
+            columns={[
+              {title: 'User', field: 'actingUserEmail'},
+              {title: 'Action', field: 'mutationType'},
+              {title: 'Object Type', field: 'objectType'},
+              {title: 'Object ID', field: 'objectId'},
+              {
+                title: 'Status',
+                field: 'status',
+                render: currRow => (
+                  <>
+                    <DeviceStatusCircle
+                      isGrey={false}
+                      isActive={currRow.status === 'SUCCESS'}
```

Comment:
Should probably group the `'SUCCESS'` with the same string in `auditLoggingDecorator.js`, into a single constant, and import it here

Reply:
I think that is a good practice, However in this case, I think 'SUCCESS' or 'FAILURE' is well understood. If this is important, then we should create a constant.js which can be imported across the board and we just use those constants across the board. To make this change for this case alone, I don't believe is useful. It might be more efficient to just grep for commonly used strings and create a single util js and move it there.